huaweicloud/config/endpoints.go

```diff
@@ -346,6 +346,11 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		ResourceBase: "notifications",
 		Product:      "SMN",
 	},
+	"smn-tag": {
```

Comment:
can we combine this with the smn endpoint above? this new endpoint will make custom endpoints on provider configuration weird.

Reply:
fixed